lib/ansible/modules/cloud/google/gcp_bigquery_dataset.py

```diff
@@ -379,14 +382,22 @@ def delete(module, link, kind):
 def resource_to_request(module):
     request = {
         u'kind': 'bigquery#dataset',
-        u'name': module.params.get('name'),
-        u'access': DatasetAccessArray(module.params.get('access', []), module).to_request(),
-        u'datasetReference': DatasetDatasetreference(module.params.get('dataset_reference', {}), module).to_request(),
-        u'defaultTableExpirationMs': module.params.get('default_table_expiration_ms'),
-        u'description': module.params.get('description'),
-        u'friendlyName': module.params.get('friendly_name'),
-        u'labels': module.params.get('labels'),
-        u'location': module.params.get('location')
+        u'name':
```

Comment:
It's these kinds of things that made format really nice. In a perfect world, I could get almost all of these on one-line.

Reply:
I wonder if we could just put them all as one-liners and then have autopep8 fix the wrong ones?

Reply:
Sort of!  We discussed this offline, but here's the code comment I made as a result of our discussion:

```py
# We are still, somewhat in the code-formatting business.  This code is
# here because autopep8 cannot handle one particular case: a function call  
# inside a function call inside a dictionary, where the first line of 
# the continuation is too long, but the second line is not. 
```

The short version, for posterity, is that, as good as `autopep8` is, it doesn't handle every case, and sometimes we're going to have to be in the code-formatting business.

We discussed alternatives - leaving `format` and `format2` as-is, for one - but rejected them, thinking that this is more explainable and less confusing for future development, accepting the downside that the logic for formatting is more spread-out.